scripts/script.yaml

```diff
@@ -352,48 +365,35 @@
       
       - name: exposures
         steps:
-          - say: מה מספר האנשים השונים (לא כולל בני ביתך) ששהית בקרבתם מעל ל-5 דקות ביממה האחרונה?
-          - wait:
-              variable: exposure_met_people
-              placehoder: מספר האנשים
-              validation: "[0-9]+"
-              input-kind: number
-
-          - say: כמה בני משפחה או דיירים אחרים מתגוררים איתך באותו הבית או באותה הדירה?
-          - wait:
-              variable: flatmates
-              placehoder: מספר האנשים
-              validation: "[0-9]+"
-              input-kind: number
-          
-          - say: כמה מהם בני 70 או יותר?
-          - wait:
-              variable: flatmates_over_70
-              placehoder: מספר האנשים
-              validation: "[0-9]+"
-              input-kind: number
-
+          - say: מה מספר האנשים השונים ששהית בקרבתם מעל ל-15 דקות בטווח 2 מטרים ביממה האחרונה?
           - say: כמה מהם מתחת לגיל 18?
           - wait:
               variable: flatmates_under_18
               plachoder: מספר האנשים
               validation: "[0-9]+"
               input-kind: number
 
-          - say: האם בעבודה יוצא לך לתת שירות ליותר מ-10 אנשים ביום?
+          - say: כמה מהם מעל לגיל 18?
```

Comment:
pointless question

Reply:
@akariv I agree, but this is the research requirement, see [the provided doc](https://docs.google.com/document/d/1eVxsBSaVIna-o932E7Ly7FAr3yhVTX7If0ZSvqXwlGQ/edit). As you probably noticed, they also canceled the "over 70" question.

Reply:
The requirement doc has two questions, and you have 3.

Reply:
I also offer 2 questions (two parts of the same question, actually). Updated the intro sentence to clarify.